BedrockServer.cpp

```diff
@@ -834,12 +834,20 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
             state = _replicationState.load();
             canWriteParallel = canWriteParallel && (state == SQLiteNodeState::LEADING);
 
+            // If the command should run prePeek, do that now and assign the result to the peekResult.
+            BedrockCore::RESULT peekResult = BedrockCore::RESULT::INVALID;
+            if (!command->repeek && !command->httpsRequests.size() && command->shouldPrePeek()) {
+                peekResult = core.prePeekCommand(command);
+                if (peekResult != BedrockCore::RESULT::SHOULD_PEEK && peekResult != BedrockCore::RESULT::SHOULD_PROCESS) {
```

Comment:
A COMPLETE result should probably be allowed as well.

I'm second guessing whether `SHOULD_PROCESS` should be allowed. Maybe we should always `peek` after `prePeek`. It makes the rest of this function easier to follow.